sample/src/main/java/org/aerogear/graphqlandroid/ConflictPojo.kt

```diff
@@ -0,0 +1,28 @@
+package org.aerogear.graphqlandroid
```

Comment:
IMHO this class is not needed. devs would not see conflict parts in their code

Reply:
This would be there in the library.